apis/common/common.go

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package v1beta1
+// +kubebuilder:object:generate=true
+// +groupName=opentelemetry.io
+package common
```

Comment:
I am concerned about using a common package for multiple versions. I haven't seen this being the case in other projects or k8s. All symbols are usually clearly versioned.

Reply:
Would it help to alias them? I don't really want to copy all the shared structs and then define pointless conversions between them.

Reply:
internal could work, but I thought that the symbols will be exposed in the CRD